packages/dds/sequence/src/test/intervalCollection.spec.ts

```diff
@@ -160,6 +165,191 @@ describe("SharedString interval collections", () => {
 			]);
 		});
 
+		describe("interval stickiness", () => {
```

Comment:
you should add stickiness generation to the interval fuzz suite (or maybe make a spinoff variant which targets stickiness especially, up to you)